### ISSUE TITLE
The private subnetwork selector is confusing

### ISSUE THREAD

**Issue by vonneudeck** [sapcc/elektra#177]
When creating a new router one has to select a private network subnet.
That interface is confusing. It looks more like an information and not like something one must select.
Maybe using something signaling that a user action is necessary, e.g. a checkbox or a dropdown would be better.
![image](https://user-images.githubusercontent.com/3678775/34716002-d87eb0fe-f52e-11e7-8ae5-645f75a6648e.png)

**Comment by edda** [comment]
Yes indeed, this is a multiselection dropbox (standard html element). But I agree that it isn't super common so people might not know how to use it. We'll think about how to make it more clear.

**Comment by andypf** [comment]
I would use javascript here and extend the standard multiselect with checkboxes

**Comment by edda** [comment]
Yeah, makes sense. We can just take an existing multiselect jquery plugin or something.
